### ISSUE TITLE
Controls: granulize "disabled" status of playback controls

### ISSUE THREAD

**Issue by 0x213F** [0x213F/jukebox-radio-react#96]
Styles will need to be updated as well.

**Comment by 0x213F** [comment]
Since this ticket was made, there have been drastic changes to the playback mechanism. The largest of such is the addition of an "playback action queue," which holds a synchronous task queue of actions that must be performed on the playback state.

For example, if we need to do a "skip to next queue" action, that gets broken down into many tasks which are pushed onto the task queue:

1: trigger pause on current playing track on Spotify
2: wait until pause is complete
3: trigger play on next YouTube track

> _Side note: the synchronous design of the queue is not optimal. Preferably, the task queue would have a dependency graph of sorts._

So anyways, this redesign has made the playback mechanism quite robust. While there are optimizations and improvements that would benefit the end user, those optimizations are not blockers. Along those lines, this ticket is not a blocker.

TL;DR: this ticket is no longer needed.
